src/DB_Command.php

```diff
@@ -176,6 +231,12 @@ public function repair() {
 	 * [--default-character-set=<character-set>]
 	 * : Use a specific character set. Defaults to DB_CHARSET when defined.
 	 *
+	 * [--dbuser=<value>]
+	 * : Username to pass to the MySQL executable. Defaults to DB_USER.
+	 *
+	 * [--dbpass=<value>]
+	 * : Password to pass to the MySQL executable. Defaults to DB_PASSWORD.
```

Comment:
`the MySQL executable` => `mysql`, to stay consistent with all of the other docblocks.